test/puppetlabs/trapperkeeper/services/authorization/authorization_service_test.clj

```diff
@@ -5,7 +5,7 @@
     [puppetlabs.ssl-utils.core :as ssl-utils]
     [puppetlabs.trapperkeeper.app :refer [get-service]]
     [puppetlabs.trapperkeeper.authorization.rules :as rules]
-    [puppetlabs.trapperkeeper.authorization.testutils :refer :all]
+    [puppetlabs.trapperkeeper.authorization.testutils :as testutils :refer :all]
```

Comment:
Might be better to just do one or the other - the `:as testutils` or the `:refer :all`.